Destiny.ScrimTracker.Logic/Models/Guardian.cs

```diff
@@ -0,0 +1,9 @@
+namespace Destiny.ScrimTracker.Logic.Models
+{
+    public class Guardian
+    {
+        public string GamerTag { get; set; }
+        public int LifetimeKills { get; set; }
+        public int LifetimeDeaths { get; set; }
```

Comment:
Should we track these over time in a similar manner to other stats?

Reply:
I figured since they will never be an up and down kinda thing, only an ever increasing number itd be fine to just track the sum. Choo thank?

Reply:
Seems fine with me unless we want to track it over time for folks?